src/app/machines/views/MachineList/MachineListTable/MachineListTable.tsx

```diff
@@ -846,8 +846,36 @@ export const MachineListTable = ({
     [hiddenColumns, showActions]
   );
 
+  const getMachinesDisplayed = () => {
```

Comment:
I was assuming this will simply be the value of currently selected pagination option (page size), but I could be wrong. https://zpl.io/z8GLgZl

Reply:
I think you might be right come to think of it, although I though it may be strange if you're on the last page and only one machine is showing, but the text says "Showing 50 out of X machines"

Reply:
Update: Asked Amy and she confirmed that this implementation (showing true number of machines on page) is correct https://chat.canonical.com/canonical/pl/4cnpzup7bjfw3p6dsa45ef4d6h